frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1124,22 +1514,8 @@ function ReaderFooter:addToMainMenu(menu_items)
             },
```

Comment:
there is a bug here

![Reader_2024-05-07_152542](https://github.com/koreader/koreader/assets/97603719/572a4b3c-fffa-4ebe-9569-d4c62b5caa07)

this option should not be showing in a non-touch device.

Reply:
@poire-z if you have time, could you look at this?

Reply:
Not super familiar with readerfooter.
Just looking and poking around for differences between frontlight and frontlight_warmth similar stuff, and the warmth one is missing what exists for the frontlight one:
https://github.com/koreader/koreader/blob/370c1e76a458f561dd00b8ff2b3b1099ae94c722/frontend/apps/reader/modules/readerfooter.lua#L496-L498
Add after this one the logical equivalent for the warmth (doesn't feel too risky, the 3 other do the same things):
```lua
    if not Device:hasNaturalLight() then
        MODE.frontlight_warmth = nil
    end
```
and your problam seems solved.

I guess you should look around that all these 4 conditionals are always handled by 4.

Reply:
Understood, thanks!